docs/root/start/quick-start/run-envoy.rst

```diff
@@ -7,6 +7,12 @@ Run Envoy
 The following instructions walk through starting Envoy as a system daemon or using
 the Envoy Docker image.
 
+.. note::
+
+   For Linux containers follow the instructions in Docker (Linux) tab and for Windows containers
```

Comment:
personally i think put this in the actual tabs - dont assume that someone reads the whole doc - also i think it needs to be clearer...

Reply:
Isnt this repetitive to have this in every tab?

Reply:
i dont think so - and personally i often ctrl-f or otherwise skip to the section im interested in so i would put it in each tab

Reply:
Do you think that a green note is distracting away someone who is reading the windows doc tab?

Reply:
perhaps dont use an actual `.. note:` - just some text would do imho

Reply:
Shouldn't we just use the title `Docker (Windows Container)` and skip the sentence. We are saving 10 characters from the tab title and then we add two sentences in each tab.

Reply:
i dont think so - pretty sure this was the conv we already had - not sure what more i can add (other than i also think that `image` is more appropriate in this context)